### PR TITLE
Add saucenao plugin

### PR DESCRIPTION
**Pull request by GospodinHoroshiy**
New plugin for searching hovered images with saucenao
Tested locally

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Plugins extend the capabilities of `nnn`. They are _executable_ scripts (or bina
 | [renamer](renamer) | Batch rename selection or files in dir [✓] | sh | [qmv](https://www.nongnu.org/renameutils/)/[vidir](https://joeyh.name/code/moreutils/) |
 | [ringtone](ringtone) | Create a variable bitrate mp3 ringtone from file | sh | date, ffmpeg |
 | [rsynccp](rsynccp) | Gives copy-paste verbose progress percentage [✓] | sh | rsync |
+| [saucenao](saucenao) | Upload file to saucenao and scrap links | sh | curl |
```
Comment (jarun) [inline plugins/README.md:60]: Can you mention that it is a reverse image search engine?

**Review by jarun — CHANGES_REQUESTED** [review]

**Comment by jarun** [comment]
Can you explain the benefit of this plugin for regular users?
Why should one use this plugin instead of opening the reverse image search engine in a web browser where she can visually see the images?

**Comment by jarun** [comment]
Closing this as I am not seeing any real use case for regular users.
Also, the script is to scrape the results which is unrelated to day-to-day file management.

**Comment by N-R-K** [comment]
A more useful version might have been to download all the images to some `/tmp/` folder and take the user there on a new context. Then they can use `nnn` to delete the entries they don't like and move/copy the ones they do.

**Comment by jarun** [comment]
@N-R-K we are not aware of the terms of usage and the copyrights of the images.

**Pull request closed without merging**